File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Model/Entities/Fascicles/FascicleDocumentModel.cs
using System;
using VecompSoftware.DocSuiteWeb.Model.Entities.Commons;
using VecompSoftware.DocSuiteWeb.Model.Entities.DocumentUnits;

namespace VecompSoftware.DocSuiteWeb.Model.Entities.Fascicles
{
    public class FascicleDocumentModel
    {
        #region [ Constructor ]


        #endregion

        #region [ Properties ]
        public Guid UniqueId { get; set; }
        public ChainType ChainType { get; set; }
        public Guid IdArchiveChain { get; set; }
        public string RegistrationUser { get; set; }
        public DateTimeOffset? RegistrationDate { get; set; }
        public string LastChangedUser { get; set; }
        public DateTimeOffset? LastChangedDate { get; set; }
        public FascicleFolderModel FascicleFolder { get; set; }
        public DocumentModel Document { get; set; }

        #endregion

        #region [ Navigation Properties ]



        #endregion
    }
}

File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/Fascicles/FascicleDocumentModel.ts
import FascicleModel = require('App/Models/Fascicles/FascicleModel');
import FascicleFolderModel = require('App/Models/Fascicles/FascicleFolderModel');
import ChainType = require('App/Models/DocumentUnits/ChainType');

interface FascicleDocumentModel {
    UniqueId: string;
    IdArchiveChain: string;
    Fascicle: FascicleModel;
    ChainType: ChainType;
    FascicleFolder: FascicleFolderModel;
    FileName: string;
    ImageUrl: string;
}

export = FascicleDocumentModel;